microcosm_pubsub/daemon.py

```diff
@@ -12,19 +14,25 @@
 
 class ConsumerDaemon(Daemon):
 
-    def __init__(self, event=None):
+    def __init__(self, enable_lambda_mode=False):
```

Comment:
since we initialize daemon before invocation, we no longer need to store sqs_event within Daemon class. enable_lambda_mode is turned ON when running in lambda mode, otherwise it is OFF by default.